src/aleph/http/client.clj

```diff
@@ -345,6 +348,21 @@
         (.remove (.pipeline ctx) this))
       (.fireUserEventTriggered ^ChannelHandlerContext ctx evt))))
 
+(defn coerce-log-level [level]
+  (if (instance? LogLevel level)
+    level
+    (let [netty-level (case level
+                        :trace LogLevel/TRACE
+                        :debug LogLevel/DEBUG
+                        :info LogLevel/INFO
+                        :warn LogLevel/WARN
+                        :error LogLevel/ERROR
+                        nil)]
```

Comment:
Just as a side note, you could thrown the exception in the default case block to avoid a local variable. Something like this:

```clojure
(defn coerce-log-level [level]
  (if (instance? LogLevel level)
    level
    (case level
      :trace LogLevel/TRACE
      :debug LogLevel/DEBUG
      :info LogLevel/INFO
      :warn LogLevel/WARN
      :error LogLevel/ERROR
      (throw (IllegalArgumentException.
               (str "unknown log level given: " level))))))
```

It also makes the code slightly shorter :-)